rules-tests/DowngradePhp81/Rector/FuncCall/DowngradeArrayIsListRector/Fixture/no_scope_parent_assign_from_func_call.php.inc

```diff
@@ -37,24 +39,24 @@ final class Loop
         if (self::$instance instanceof LoopInterface) {
             return self::$instance;
         }
+        $arrayIsList = function (array $array) : bool {
+            if (function_exists('array_is_list')) {
+                return array_is_list($array);
+            }
+            if ($array === []) {
+                return true;
+            }
+            $current_key = 0;
+            foreach ($array as $key => $noop) {
+                if ($key !== $current_key) {
+                    return false;
+                }
+                ++$current_key;
+            }
+            return true;
+        };
 
         register_shutdown_function(function () {
-            $arrayIsList = function (array $array) : bool {
-                if (function_exists('array_is_list')) {
-                    return array_is_list($array);
-                }
-                if ($array === []) {
-                    return true;
-                }
-                $current_key = 0;
-                foreach ($array as $key => $noop) {
-                    if ($key !== $current_key) {
-                        return false;
-                    }
-                    ++$current_key;
-                }
-                return true;
-            };
             $test = $arrayIsList([]);
```

Comment:
@TomasVotruba the variable `$arrayIsList` is undefined here, and must be passed via Closure use to be available, I will look into it.